easybuild/easyconfigs/d/DCA/DCA++-CMake-CrayGNU-2015.11.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
+
+name = 'DCA++'
+version = '1.0'
```

Comment:
You could also add 
```
versionsuffix = '-cuda' 
```
to make the dependency on gpu explicit (or `-gpu`). As opposed to the `-dora` that actually means cpu-only.